RetrofitLab/app/src/main/java/net/serkanbal/retrofitlab/OpenAirWeatherService.java

```diff
@@ -0,0 +1,16 @@
+package net.serkanbal.retrofitlab;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+/**
+ * Created by Serkan on 18/11/16.
+ */
+
+public interface OpenAirWeatherService {
+
+@GET("weather?appid=1dc7f58282bec421f230ea3985ee589d&units=metric")
```

Comment:
Best practice would be to make `appid` and `units` parameters with the `@Query` annotation, like you did with `q` below - also make the appid value a constant somewhere